### DISCUSSION TITLE
Is it possible to reuse param_scheduler on loss weight?

### DISCUSSION THREAD

**Discussion by twmht** [open-mmlab/mmengine#937] [Ideas]
Hi,

I would like to use `param_scheduler`  on loss_weight, in some algorithms, we may want to ramp up the loss weight until the certain value. but in the current implementations, we can only pass `optimi_wrapper` to  `param_scheduler`.  So it would be better to generlize that.

Any idea?

**Comment by zhouzaida** [comment]
Hi, can we implement the feature using a hook to warm up the loss weight?

**Comment by twmht** [comment]
@zhouzaida 

yup sure, but how can we implement without modifying the current `param_scheduler `?
